docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -155,17 +157,41 @@ throughout this style guide.
    * - Term
      - Definition
    * - Sex
-     - *Sex* refers to biological attributes and functions associated with chromosomal makeup, reproductive anatomy, and secondary sex characteristics [Ritz_and_Greaves_2022]_.
+     - *Sex* refers to biological attributes and functions associated with chromosomal makeup, 
+       reproductive anatomy, and secondary sex characteristics [Ritz_and_Greaves_2022]_.
    * - Gender
-     - *Gender*, on the other hand, is a social construct that refers to the culturally specific expectations, behaviors, and institutions that shape any individual's access to power, privilege, and resources. It can also refer to a gendered individual's internal sense of identity [Ritz_and_Greaves_2022]_. 
+     - *Gender*, on the other hand, is a social construct that refers to the culturally specific 
+       expectations, behaviors, and institutions that shape any individual's access to power, 
+       privilege, and resources. It can also refer to a gendered individual's internal sense of 
+       identity [Ritz_and_Greaves_2022]_. 
    * - Sexual and gender minority (SGM) 
-     - An umbrella term that includes (but is not limited to) individuals that are two-spirit, intersex, lesbian, gay, bisexual, transgender, non-binary, asexual, or have other gender non-conforming identities.
+     - An umbrella term that includes (but is not limited to) individuals that are two-spirit, 
+       intersex, lesbian, gay, bisexual, transgender, non-binary, asexual, or have other gender 
+       non-conforming identities.
    * - Undifferentiated sex/gender measure
-     - A useful hybrid term that reflects the dynamic relationship that exists between sex and gender and acknowledges the common conflation between sex and gender in public health research and data [Ritz_and_Greaves_2022]_. 
+     - Due to data limitations or availability, it might not always be possible to elucidate whether a
+       certain data source is intended to capture sex or gender, or some factor related to both. In such 
+       cases, we will capture this uncertainty by using an *undifferentiated sex/gender measure*, a binary
+       variable that combines sex and gender measures across data sources. See Section `2.0 Dealing with 
```

Comment:
Can we flip the sentence structure to lead with the term? 

*Undifferentiated sex/gender measure* is a binary variable that combines sex and gender measures across data sources. Due to data limitations or availability, it might not always be possible to elucidate whether a certain data source is intended to capture sex or gender, or some factor related to both. This term captures that uncertainty.